### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by leonardoeng13**
# Mobile Menu Function.

### In mobile, implement the following features:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,7 +27,20 @@
...
+                        <div class="line line-1"></div>
+                        <div class="line line-2"></div>
+                        <div class="line line-3"></div>
+                    </div>
+                    <ul class="hamb-nav-list">
+                        <li class="hamb-nav-item"><a href="#mobile-index" target="_self" class="nav-link">Hello</a></li>
+                        <li class="hamb-nav-item"><a href="#mobileportfolio" target="_self" class="nav-link">Portfolio</a></li>
+                        <li class="hamb-nav-item"><a href="#mobileabout" target="_self" class="nav-link">About</a></li>
+                        <li class="hamb-nav-item"><a href="#mobilecontact" target="_self" class="nav-link">Contact</a></li>
+                    </ul>
```
Comment (vichuge) [inline index.html]: - [x] When I click the options, the menu doesn't disappear, please, work on this. Remember the menu should disappear (or close).
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] It looks like you're using js linters config, but no HTML-CSS-js linters config, I kindly invite you to change it according to the [template](https://github.com/microverseinc/linters-config/tree/master/html-css-js). 🐶 
image
- [ ] It looks like you have an issue with responsiveness, I invite you to work on that too. It's hard to test (or including use) the mobile menu without having a good responsive design. 🐱
image

- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (leonardoeng13) [reply index.html]: Fixed.

File: index.html [hunk 2]
```diff
@@ -27,7 +27,20 @@
                 <a href="#"><img src="images/logomobile.svg" alt="Logo Mobile"></a>
             </div>
             <div class="menumobile">
-                <a href=""><img src="images/ic_menu.svg" alt="Mobile Menu"></a>
+                <nav class="navbar-mobile">
+                    <div class="hamburguer-menu">
+                        <div class="line line-1"></div>
+                        <div class="line line-2"></div>
+                        <div class="line line-3"></div>
+                    </div>
```
Comment (arslanbisharat) [inline index.html:35]: - [ ] I cannot see the hamburger on the mobile version. I tried different browsers too. Please troubleshoot this issue.  
Capture

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- [ ]  [Optional] Please remove `.vscode ` from project directory. You can add it to your `gitignore `file.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 2 above)
Comment (leonardoeng13) [reply index.html:35]: You have to use the "mobile" version of your navigator. 

image

**Comment by arslanbisharat** [comment]
Hi there,
I am sorry for this inconvenience. Please request another code review. 🙏

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

I'm happy that you fixed the issue, and got it approved
For future projects, please make sure to either leave a note about the website only working using dev tools or in the CSS make it so that all devices would change view even if it was a PC :)

Good luck with future projects, Keep rocking 🚀!

**Pull request merged**